8.cytopick_analysis/scripts/prediction_images_find.py

```diff
@@ -483,17 +498,62 @@ def random_cell_select(
             red_channel = (
                 red_channel_stack / np.max(red_channel_stack) * 65535
             ).astype(np.uint16)
+
+            # blue_channel_crop = (
+            #     blue_channel_stack_crop / np.max(blue_channel_stack_crop) * 65535
+            # ).astype(np.uint16)
+            # green_channel_crop = (
+            #     green_channel_stack_crop / np.max(green_channel_stack_crop) * 65535
+            # ).astype(np.uint16)
+            # red_channel_crop = (
+            #     red_channel_stack_crop / np.max(red_channel_stack_crop) * 65535
+            # ).astype(np.uint16)
+
+            # merge the channels together
+
             composite_image = cv2.merge(
-                (blue_channel, green_channel, red_channel)
+                (red_channel, green_channel, blue_channel)
             ).astype(np.uint16)
+            # composite_image = cv2.cvtColor(composite_image, cv2.COLOR_BGR2RGB)
+
+            composite_image_crop = composite_image[
+                min_y_box:max_y_box, min_x_box:max_x_box
+            ]
+
+            # im_crop = composite_image[min_y_box:max_y_box, min_x_box:max_x_box]
+            if composite_image_crop.shape[0] == 0 or composite_image_crop.shape[1] == 0:
+                print("Cell is on the edge of the image, skipping")
+                continue
+
+            # composite_image_crop = cv2.merge(
+            #     (blue_channel_crop, green_channel_crop, red_channel_crop)
+            # ).astype(np.uint16)
             composite_image = cv2.cvtColor(composite_image, cv2.COLOR_BGR2RGB)
+            composite_image_crop = cv2.cvtColor(composite_image_crop, cv2.COLOR_BGR2RGB)
 
+            # The images end up being `wonky` so we need to do some post processing prior to saving
```

Comment:
Consider documenting `wonky` further, i.e. how?

Reply:
Added documentation to describe how